README.md

```diff
@@ -157,6 +157,8 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
    #### Install with Apple Metal on M1/M2/M3 Macs
 
+      > **NOTE**: Make sure your system Python build is `Mach-O 64-bit executable arm64` by using `file -b $(command -v python)`.
```

Comment:
Not worth blocking on but FYI there’s a nicer way to render NOTEs —

https://github.com/orgs/community/discussions/16925